### PR TITLE
Standardise auth variables

### PR DESCRIPTION
**Pull request by mjfwebb**
- [use "username" in where username would be used in code examples](https://github.com/neo4j/docs-graphql/commit/9c27e9cd312894a317a39d3f39f11d3665fcb5b4)
- [fix typos](https://github.com/neo4j/docs-graphql/commit/9c06239b6bc93cd1726140eacf627d22c5671fb4)
- [remove part about creating a .env file](https://github.com/neo4j/docs-graphql/commit/e256b9175c8e76db8f9dc431a70e1b673825fe60)

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: modules/ROOT/pages/integrations/apollo-federation.adoc [hunk 1]
```diff
@@ -20,16 +20,11 @@ const typeDefs = `#graphql
     name: String!
   }
 `;
+;
```
Comment (lidiazuin) [inline modules/ROOT/pages/integrations/apollo-federation.adoc]: Does it need a new ; or is this a typo?

**Review by lidiazuin — APPROVED** [review]
TY Michael!

File: modules/ROOT/pages/integrations/apollo-federation.adoc (hunk 1 above)
Comment (mjfwebb) [reply modules/ROOT/pages/integrations/apollo-federation.adoc]: Nice catch!

**Pull request merged**